code/datums/gamemode/factions/bloodcult/bloodcult_mobs_and_constructs.dm

```diff
@@ -0,0 +1,439 @@
+
+/////////////////Juggernaut///////////////
+/mob/living/simple_animal/construct/armoured/perfect
+	icon_state = "juggernaut2"
+	icon_living = "juggernaut2"
+	icon_dead = "juggernaut2"
+	construct_spells = list(
+		/spell/aoe_turf/conjure/forcewall/greater,
+		/spell/juggerdash,
+		)
+	see_in_dark = 7
+	var/dash_dir = null
+	var/turf/crashing = null
+
+/mob/living/simple_animal/construct/armoured/perfect/New()
+	..()
+	setupfloat()
+
+/mob/living/simple_animal/construct/armoured/perfect/to_bump(var/atom/obstacle)
+	if(src.throwing)
+		var/breakthrough = 0
+		if(istype(obstacle, /obj/structure/window/))
+			obstacle.Destroy(brokenup = 1)
+			breakthrough = 1
+
+		else if(istype(obstacle, /obj/structure/grille/))
+			var/obj/structure/grille/G = obstacle
+			G.health = (0.25*initial(G.health))
+			G.broken = 1
+			G.icon_state = "[initial(G.icon_state)]-b"
+			G.setDensity(FALSE)
+			getFromPool(/obj/item/stack/rods, get_turf(G.loc))
+			breakthrough = 1
+
+		else if(istype(obstacle, /obj/structure/table))
+			var/obj/structure/table/T = obstacle
+			T.destroy()
+			breakthrough = 1
+
+		else if(istype(obstacle, /obj/structure/rack))
+			new /obj/item/weapon/rack_parts(obstacle.loc)
+			qdel(obstacle)
+			breakthrough = 1
+
+		else if(istype(obstacle, /turf/simulated/wall))
+			var/turf/simulated/wall/W = obstacle
+			if (W.hardness <= 60)
```

Comment:
Pray tell what this magical number means

Reply:
![image](https://user-images.githubusercontent.com/7573912/46963323-a6b71e00-d0a5-11e8-89ef-2faa017bb978.png)

Basically anything above 60 is reinforced.